packages/Utils/src/DTK_Core.hpp

```diff
@@ -0,0 +1,28 @@
+/****************************************************************************
+ * Copyright (c) 2012-2017 by the DataTransferKit authors                   *
```

Comment:
Mostly i was thinking that when we are done with everything the user would only have to include a single header file to use DTK in C